spinedb_api/db_mapping_base.py

```diff
@@ -556,7 +556,7 @@ def remove_item(self, item):
             self.purged = True
             for current_item in self.valid_values():
                 self.remove_unique(current_item)
-                current_item.cascade_remove(source=self.wildcard_item)
```

Comment:
This breaks this test in spinetoolbox: tests.spine_db_editor.widgets.test_custom_qtableview.TestParameterTableWithExistingData.test_rolling_back_purge

Reply:
I think the problem is we don't rollback purge, see here https://github.com/spine-tools/Spine-Database-API/issues/357

This is good I think.

Reply:
We do rollback purge. The issue in the end is we were still passing the wildcard_item as source to cascade_remove in the lazy purge (the purge that happens at fetch time). I think I fixed it now. Good work.

Reply:
Thanks for fixing the issue!